etc/greenboot/green.d/00_grub_fallback.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+# We need to determine whether this boot was a fallback boot
+if grub2-editenv list | grep -q "fallback_active=true"; then
+  grub2-editenv - unset fallback_active
```

Comment:
we could as well leave this set until next rpm-ostree upgrade

Reply:
I think leaving it set makes more sense yeah. We can use that information elsewhere as well to let users know this is a fallback boot.